src/main/java/beans/UserBean.java

```diff
@@ -0,0 +1,140 @@
+package beans;
+
+import java.io.Serializable;
+
+public class UserBean implements Serializable {
+	
+	public static final long serialVersionUID = 1L;
+	
+	private int userId;
+	private boolean gender;
+	private String dob;
+	private String name;
+	private String surname;
+	private String username;
+	private String email;
+	private String salt;
+	private String password;
+	private String description;
+	
+	public UserBean () {}
+	
+	public UserBean(int userID,
+			String name,
+			String surname,
+			String username,
+			String email,
+			String salt,
+			String password,
+			boolean gender, 
+			String dob, 
+			String description) {
+		super();
+		this.userId = userID;
+		this.name = name;
+		this.surname = surname;
+		this.username = username;
+		this.email = email;
+		this.salt = salt;
+		this.password = password;
+		this.gender = gender;
+		this.dob = dob;
+		this.description = description;
+	}
+	
+	private String resolveNull(String string) {
```

Comment:
this can be achieved with a single line for more readability (string == null ? "": string)